cloud/backend/cf-d1/cf-hono-server.ts

```
@@ -40,7 +38,9 @@ function webSocket2WSContextInit(ws: WebSocket): WSContextInit<WebSocket> {
     send: (data: string | ArrayBuffer, _options: SendOptions): void => {
       ws.send(data);
     },
-    close: (code?: number, reason?: string): void => ws.close(code, reason),
+    close: (code?: number, reason?: string): void => {
+      return ws.close(code, reason);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)